Description: when inline functions are implemented not where they are
 declared, linking fails. Make statrep() and lstatrep() non-inline to
 fix this.
Author: Ilya Barygin <barygin@gmail.com>

--- xfe-1.32.1.orig/src/xfeutils.cpp
+++ xfe-1.32.1/src/xfeutils.cpp
@@ -512,7 +512,7 @@ FXbool isUtf8(const FXchar* string, FXui
 
 
 // Replacement of the stat function
-inline FXint statrep(const FXchar* filename, struct stat* buf)
+FXint statrep(const FXchar* filename, struct stat* buf)
 {
 #if defined(linux)
 
@@ -546,7 +546,7 @@ inline FXint statrep(const FXchar* filen
 
 
 // Replacement of the lstat function
-inline FXint lstatrep(const FXchar* filename, struct stat* buf)
+FXint lstatrep(const FXchar* filename, struct stat* buf)
 {
 #if defined(linux)
 
